data/allowed_package_sources.yml

```
@@ -0,0 +1,6 @@
+---
+rule_data:
+  allowed_package_sources:
+    - type: generic
```

```suggestion
    - type: maven
```

Hey, cachi2 now [supports](https://github.com/containerbuildsystem/cachi2/blob/main/docs/generic.md#maven-artifacts) fetching maven artifacts directly and reporting them as such in the SBOM. If users used the generic functionality to download maven artifacts, they should migrate to the maven ones, as the indy.corp.redhat.com only hosts those.

---

Please reach out to @dchourasia and @MohammadiIram to discuss this

---

As for narrowing the url, cc @aloubyansky